packages/text-editor/src/components/ReferenceInput.svelte

```diff
@@ -156,195 +157,133 @@
   })
 </script>
 
-<div class="ref-container">
-  <div class="textInput" class:withoutTopBorder>
-    <div class="inputMsg">
-      <TextEditor
-        bind:content
-        bind:this={textEditor}
-        on:content={(ev) => {
-          if (!isEmpty || haveAttachment) {
-            dispatch('message', ev.detail)
-            content = ''
-            textEditor.clear()
-          }
-        }}
-        on:blur={() => {
-          focused = false
-          dispatch('blur', focused)
-        }}
-        on:focus={() => {
-          focused = true
-          updateFocus()
-          dispatch('focus', focused)
-        }}
-        extensions={[
-          completionPlugin,
-          EmojiExtension.configure(),
-          IsEmptyContentExtension.configure({ onChange: (value) => (isEmpty = value) })
-        ]}
-        on:update
-        placeholder={placeholder ?? textEditorPlugin.string.EditorPlaceholder}
-        textFormatCategories={[
-          TextFormatCategory.TextDecoration,
-          TextFormatCategory.Link,
-          TextFormatCategory.List,
-          TextFormatCategory.Quote,
-          TextFormatCategory.Code
-        ]}
-      />
+<div class="ref-container" class:focusable={kind === 'accented'}>
+  {#if showHeader && $$slots.header}
```

Comment:
Why do we need `showHeader` here? Isn't missing `header` in `$$slots` already provides the same condition?

Reply:
It is mostly to manage bottom border and paddings for header component, I think it is better to manage it inside the component itself than in slot